templates/analytics/remote_server_support.html

```diff
@@ -88,6 +88,26 @@ <h4>On 100% sponsored Zulip Community plan.</h4>
                 {% endif %}
             </form>
             {% endif %}
+
+            {% if plan_data[remote_server.id].current_plan %}
+            <div class="remote-server-information">
+                <h4>📅 Current plan information:</h4>
+                <b>Plan name</b>: {{ plan_data[remote_server.id].current_plan.name }}<br />
+                <b>Status</b>: {{plan_data[remote_server.id].current_plan.get_plan_status_as_text()}}<br />
+                {% if plan_data[remote_server.id].current_plan.tier != 101 %}
```

Comment:
Let's not hardcode values like this 101 into the template (or maybe just have the caller compute a boolean field to pass in) -- much better to pass a variable in so that if we a renumbering, we'll update this; also make grepping much happier/